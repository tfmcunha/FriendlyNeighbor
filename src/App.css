.App {
  text-align: center;
}

body {
  padding-bottom: 20px
}