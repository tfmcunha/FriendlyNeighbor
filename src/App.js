import React, { Component } from 'react';
import { Container } from 'react-bootstrap';
import Main from './components/main';


class App extends Component {
  render() {
    return (        
      <Container>        
        <Main />
      </Container>
    );
  }
}

export default App;
