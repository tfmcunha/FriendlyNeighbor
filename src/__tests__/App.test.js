import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import {MemoryRouter} from 'react-router';

import App from '../App';

describe('<App />', () => {
	it('matches the snapshot', () => {
		const tree = renderer.create(<MemoryRouter><App /></MemoryRouter>).toJSON()
		expect (tree).toMatchSnapshot()
	})
})