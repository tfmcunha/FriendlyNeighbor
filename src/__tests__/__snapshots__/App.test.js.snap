// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> matches the snapshot 1`] = `
<div
  className="container"
>
  <div
    className="mb-5"
  >
    <div
      className="cover"
    >
      <div
        className="row"
      >
        <div
          className="d-none d-lg-block col-lg-6"
        >
          <div
            className="d-flex justify-content-center align-items-center vh-100"
          >
            <div
              className="m-5 box"
            >
              <h1
                className="display-4 p-4 text-center"
              >
                JOIN YOUR NEIGHBOURS IN A NETWORK OF MUTUAL AID!
              </h1>
              <div
                className="text-center m-2"
              >
                <h3>
                  Unfulfilled help requests
                </h3>
                <h1>
                  
                </h1>
              </div>
            </div>
          </div>
        </div>
        <div
          className="col-lg-6"
        >
          <div
            className="d-flex justify-content-center align-items-center vh-100"
          >
            <div
              className="p-2 box"
            >
              <div
                className="m-2 brand text-center"
              >
                 Friendly Neighbour 
              </div>
              <img
                className="d-block img-fluid"
                src="./handyman.png"
              />
              <div
                className="d-flex justify-content-center"
              >
                <button
                  className="m-2 btn btn-primary"
                  disabled={false}
                  onClick={[Function]}
                  type="button"
                >
                  Login
                </button>
                <button
                  className="m-2 btn btn-primary"
                  disabled={false}
                  onClick={[Function]}
                  type="button"
                >
                  Register
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div>
      <nav
        className="myfooter justify-content-center navbar navbar-expand navbar-light fixed-bottom"
      >
        <div
          className="mr-1 navbar-nav"
          onKeyDown={[Function]}
        >
          <div
            className="social nav-item"
          >
            <a
              className="nav-link"
              href="/help"
              onClick={[Function]}
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 512 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M504 256c0 136.997-111.043 248-248 248S8 392.997 8 256C8 119.083 119.043 8 256 8s248 111.083 248 248zM262.655 90c-54.497 0-89.255 22.957-116.549 63.758-3.536 5.286-2.353 12.415 2.715 16.258l34.699 26.31c5.205 3.947 12.621 3.008 16.665-2.122 17.864-22.658 30.113-35.797 57.303-35.797 20.429 0 45.698 13.148 45.698 32.958 0 14.976-12.363 22.667-32.534 33.976C247.128 238.528 216 254.941 216 296v4c0 6.627 5.373 12 12 12h56c6.627 0 12-5.373 12-12v-1.333c0-28.462 83.186-29.647 83.186-106.667 0-58.002-60.165-102-116.531-102zM256 338c-25.365 0-46 20.635-46 46 0 25.364 20.635 46 46 46s46-20.636 46-46c0-25.365-20.635-46-46-46z"
                />
              </svg>
            </a>
          </div>
          <div
            className="social nav-item"
          >
            <a
              className="nav-link"
              data-rb-event-key="mailto:tcunha_lp@hotmail.com"
              href="mailto:tcunha_lp@hotmail.com"
              onClick={[Function]}
              onKeyDown={[Function]}
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 512 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"
                />
              </svg>
            </a>
          </div>
          <div
            className="social nav-item"
          >
            <a
              className="nav-link"
              data-rb-event-key="http://www.facebook.com"
              href="http://www.facebook.com"
              onClick={[Function]}
              onKeyDown={[Function]}
              target="blank"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 448 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M448 56.7v398.5c0 13.7-11.1 24.7-24.7 24.7H309.1V306.5h58.2l8.7-67.6h-67v-43.2c0-19.6 5.4-32.9 33.5-32.9h35.8v-60.5c-6.2-.8-27.4-2.7-52.2-2.7-51.6 0-87 31.5-87 89.4v49.9h-58.4v67.6h58.4V480H24.7C11.1 480 0 468.9 0 455.3V56.7C0 43.1 11.1 32 24.7 32h398.5c13.7 0 24.8 11.1 24.8 24.7z"
                />
              </svg>
            </a>
          </div>
          <div
            className="social nav-item"
          >
            <a
              className="nav-link"
              data-rb-event-key="http://www.twitter.com"
              href="http://www.twitter.com"
              onClick={[Function]}
              onKeyDown={[Function]}
              target="blank"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 512 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
                />
              </svg>
            </a>
          </div>
          <div
            className="social nav-item"
          >
            <a
              className="nav-link"
              data-rb-event-key="http://www.instagram.com"
              href="http://www.instagram.com"
              onClick={[Function]}
              onKeyDown={[Function]}
              target="blank"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                strokeWidth="0"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 448 512"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
                />
              </svg>
            </a>
          </div>
        </div>
      </nav>
    </div>
  </div>
</div>
`;
